src/scripts/request.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
+set -o pipefail
+set -e
+
+if [[ $EUID == 0 ]]; then export SUDO=""; else # Check if we are root
+  export SUDO="sudo";
+fi
+
+if ! command -v envsubst >/dev/null 2>&1; then
+  $SUDO apt-get -qq -y update
+  $SUDO apt-get -qq -y install gettext-base
+fi
 
-sudo apt-get update
-sudo apt-get install gettext-base
 echo "========================================================================"
 FLOW_SRN=$(echo $FLOW_SRN | envsubst)
 FLOW_INPUTS=$(echo $FLOW_INPUTS | envsubst)
 CONTEXT=$(echo $CONTEXT | envsubst)
-REQUEST_SLUG=$(echo $REQUEST_SLUG | envsubst)
```

Comment:
dont envsubst REQUEST_SLUG for consistency w/the other cmds

Reply:
or should we go the other direction where we `envsubst` `REQUEST_SLUG` everywhere? so that way you could do something like use the `$CIRCLE_SHA1 ` for the request slug or something if you wanted to? 🤷

Reply:
Well it takes 10 seconds or so to install it, i was thinking lets avoid this in jobs unless we really need it.